test/infrastructure/docker/Makefile

```diff
@@ -47,7 +47,7 @@ endif
 # Binaries.
 CONTROLLER_GEN := $(TOOLS_BIN_DIR)/controller-gen
 CONVERSION_GEN := $(TOOLS_BIN_DIR)/conversion-gen
```

Comment:
GOLANGCI_LINT was unused. Probably because the top-level golangci-lint target lints the whole repo